devices.js

```diff
@@ -56,6 +56,18 @@ const readMeteringPowerConverterAttributes = async (endpoint) => {
     await endpoint.read('seMetering', ['multiplier', 'divisor']);
 };
 
+const writeCurrentTime = async (endpoint) => {
```

Comment:
Same as https://github.com/Koenkk/zigbee-herdsman-converters/pull/1564/files#r491344969

Reply:
I thing this can be useful for other devices. Heiman and may be other brands. It is standard cluster and standard writing to it. 
I'm sure that at least Heiman will release some new device and this will be needed for it.